lib/api.js

```diff
@@ -2,15 +2,30 @@ import fs from 'fs'
 import { join } from 'path'
 import matter from 'gray-matter'
 
-const postsDirectory = join(process.cwd(), 'content/posts')
+/**
+ * @param {string} locale
+ * @returns string
+ */
```

Comment:
These comments are JSDoc comments that give more context to IDEs about what params and return values are expected. Provides better autocomplete and highlights potential problems with unexpected params.

Obviously feel free to remove.